extensions/wikia/Discussions/maintenance/StaffWelcomePoster.class.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+class StaffWelcomePoster {
+
+	const LANG_TO_STAFF_MAP = [
+		'de' => 24961680, // ForestFairy
+		'en' => 26339491, // Mira Laime
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)